src/waypoint.jsx

```diff
@@ -333,13 +352,20 @@ export default class Waypoint extends React.Component {
    * @return {Object}
    */
   render() {
-    // We need an element that we can locate in the DOM to determine where it is
-    // rendered relative to the top of its context.
-    return <span ref={this.refElement} style={{ fontSize: 0 }} />;
+    if (this.props.children) {
+      // TODO: consider returning children directly when there is only one child,
+      // using React.cloneElement to attach a ref (but child might already have a ref)
```

Comment:
I understand the desire to keep the number of nodes small. I wonder if returning `children` directly would require a refactoring of `_getBounds` to account for an array of children, as we wouldn't be able to rely on a single element's `getBoundingClientRect`?

In any event, I agree with @trotzig that this is good as-is :+1: